src/Sentry/Protocol/Attachment.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Sentry.Protocol
+{
+    /// <summary>
+    /// Attachment type.
+    /// </summary>
+    public enum AttachmentType
+    {
+        /// <summary>
+        /// Standard attachment without special meaning.
+        /// </summary>
+        Default,
+
+        /// <summary>
+        /// Minidump file that creates an error event and is symbolicated.
+        /// The file should start with the <code>MDMP</code> magic bytes.
+        /// </summary>
+        MiniDump,
```

Comment:
Minidump is one word:

```suggestion
        Minidump,
```